cl/search/feeds.py

```diff
@@ -97,6 +97,38 @@ def item_title(self, item):
     def item_categories(self, item):
         return [item['status'], ]
 
+    def item_enclosure_url(self, item):
+        try:
+            path = item['local_path']
+            if not path.startswith('/'):
+                return '/%s' % (path,)
```

Comment:
Gotcha. This actually makes me think that we should use `reverse` here too, reversing the URL for `serve` static file. That should fix this issue, making it explicit how it works, and also allow us to change the URL down the road if we need to.